src/scss/blocks/_figure.scss

```diff
@@ -2,34 +2,35 @@
 /// https://web.dev/design-system/component/figure
 figure {
   border: 1px solid var(--color-stroke);
-  padding: 1em;
   border-radius: $global-radius-base;
+  display: inline-block;
+  padding: 1em;
+
```

Comment:
figure needs to be centered in the prose. right now it is inlined to the left sode.

Reply:
I'd be totally fine with that. I don't think the border is the most important thing. It isn't really used consistently for me to think it's a good design choice either... =/